src/node_options.cc

```diff
@@ -129,6 +129,12 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "show stack traces on process warnings",
             &EnvironmentOptions::trace_warnings,
             kAllowedInEnvironment);
+#if defined(NODE_REPORT)
+  AddOption("--report-events",
```

Comment:
The naming on this is a bit odd and misleading. Why not just `--node-report` to enable?

Reply:
subsequent reviews and discussions evolved this into `--experimental-report`